backend/api/controllers/PullRequestController.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const PullRequest = require('../../services/PullRequest')
+
+const PullRequestController = express()
+
+PullRequestController.get('/:year?', async function (req, res) {
```

Comment:
What the question mark does? :thinking:

Reply:
It basically makes the param optional. So, if someone does not enter a year, they will be given the results from that year.